distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -350,13 +346,6 @@ public void tearDown() throws Exception {
 
         getSecurityPolicy().configureRestForGuest();
 
-        // @formatter:off
```

Comment:
Now that I look at the tests, I think we need to keep and update this code. We should try to see if we can change the CometD client to capture all the download IDs and add a method on it to cancel all the active downloads. This should allow us to make sure all downloads are cancelled before we move on to the next test.